people.md

```diff
@@ -25,11 +25,15 @@ postdoc:
         [<i class='fab fa-linkedin fa-2x'></i>](https://www.linkedin.com/in/dwroush/)&nbsp;&nbsp;
         [<i class='ai ai-cv-square ai-2x'></i>](assets/files/roush-cv-aug-2020-qiyun-lab.pdf)
   - title: Julian Yu
-    image_path: assets/images/people/blank.png
+    image_path: assets/images/people/Yu-labphoto.JPG
     excerpt: >
-      I am Julian Yu.
+      I am a postdoctoral scholar under Dr. Qiyun Zhu at the Center for Fundamental and Applied Microbiomics. My interest in soil microbiomes propelled me to complete my Ph.D under Dr. Christopher Ryan Penton at Arizona State University. My dissertation research examined the taxonomic and functional shifts in the  microbial communities of agricultural soils that had received biochar application. Currently, my research interest include employing 'omics tools to understand the mechanisms underlying the microbial interactions between taxa and their environment. 
```

Comment:
@yujulian0168 Good job! I have some thoughts. Feel free to comment!

I wouldn't recommend saying "_under_ Dr. Qiyun Zhu". We are a team! This is obvious in this webpage. No need to spell that out. If you do want to state this (in other situations), please say "I am working _with_ Dr. Qiyun Zhu".

How about adding "developing" before "employing"? Using (other people's) tools does not sound super novel. I expect that your bioinformatics skills will be much better than just doing pipelining.

"My dissertation research examined" => "During my dissertation research I examined"

"in the  microbial communities of": Redundant whitespace.

"my research interest include" => "...interests..."

"underlying the microbial interactions between taxa...": Duplicated. How about "the intereactions between microbial taxa..."

Reply:
Thank you for the comments. I have made the suggested revisions.